client/src/components/Model.js

```diff
@@ -264,6 +367,26 @@ export default class Model {
     return a && b && a.uuid === b.uuid
   }
 
+  static fetchByUuid(uuid, ENTITY_GQL_FIELDS) {
```

Comment:
Function parameters should be camelCase (UPPERCASE_UNDERSCORED is for constants).